projects/storefrontlib/src/cms-components/cart/add-to-cart/add-to-cart.component.ts

```diff
@@ -43,16 +55,41 @@ export class AddToCartComponent implements OnInit, OnDestroy {
     quantity: new FormControl(1, { updateOn: 'blur' }),
   });
 
+  constructor(
+    modalService: ModalService,
+    currentProductService: CurrentProductService,
+    cd: ChangeDetectorRef,
+    activeCartService: ActiveCartService,
+    componentData: CmsComponentData<model>
+  );
+
+  /**
+   * @deprecated since 3.4
+   */
+  constructor(
```

Comment:
Don't need to do the deprecation on the file since this is going to 4.0 correct? 

What you need to do is to include it in the migration file and schematics (4.0.md) and constructor-deprecation.stuff

Reply:
Talked with Bill and he told me inventory display will be included in 4.x. Will I need to include the migration and schematic files despite it not being for 4.0??

Reply:
Since this is late for 4.0, you said Marcin wants this in 4.1 now